### ISSUE TITLE
Expose "Sykkelhotell" better

### ISSUE THREAD

**Issue by tsobuskerudbyen** [buskerudbyen/cycling-norway#30]
Similar to the Bikely solutions, can we expose information about the solution from Bane NOR Eiendom: "Sykkelhotell". They are alle controlled by a single app.
Show

1. name
2. capacity 
3. Website as an link. For security, should it be hard coded?

Green background on the capacity number indicates that there are some free space. Can the background be in a natural colour?

Example: 
https://www.openstreetmap.org/way/1002161066

**Comment by leonardehrenfried** [comment]
Yes, that can be done. @tsobuskerudbyen I've moved it into Rebeka's Todo column.

(The complete board is available here: https://github.com/orgs/buskerudbyen/projects/1/)

**Comment by Beck-berry** [comment]
@tsobuskerudbyen I also change the minZoom level to 15 here, is that ok?

**Comment by tsobuskerudbyen** [comment]
Looks good. Thanks.
